src/utils/html.rs

```diff
@@ -0,0 +1,28 @@
+use std::string::String;
+
+// Escapes the string to be shown "as is" within the Telegram HTML message style.
+// Does not escape ' and " characters (as should be for usual HTML).
+// Because they shoudn't be escaped by the spec: https://core.telegram.org/bots/api#html-style
```

Comment:
```suggestion
/// Escapes the string to be shown "as is" within the Telegram HTML message style.
///
/// Does not escape ' and " characters (as should be for usual HTML), because they shoudn't
/// be escaped by the [spec].
///
/// [spec]: https://core.telegram.org/bots/api#html-style
```